src/js/wrap.cpp

```diff
@@ -90,17 +115,32 @@ namespace ccf::js
     }
     size_t stack_size = 1024 * 1024;
     size_t heap_size = 100 * 1024 * 1024;
+    std::chrono::seconds default_max_execution_time{240};
```

Comment:
This default value is still up in the air

Reply:
I think we need finer granularity than seconds (probably milliseconds), and that the default should be `O(10s)` as an absolute max for now, and I think that 100ms might be more sensible. Since we're still single threaded, a single endpoint executing for this time is blocking the node from doing any other work. This includes making it essentially dead as far as consensus is concerned.

Reply:
>  This includes making it essentially dead as far as consensus is concerned.

That's a good point. I will test this with lower values

Reply:
`100ms` is too low, the npm app test itself is failing. I've updated the default to `1s`